ci/linux-debian.Dockerfile

```diff
@@ -23,6 +25,29 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
         sagemath
 
 WORKDIR /root
+
+# Install gcc snapshot
+RUN wget --progress=dot:giga --content-disposition https://kayari.org/gcc-latest/gcc-latest.deb && \
+    dpkg -i gcc-latest_*.deb && \
+    ln -s /opt/gcc-latest/bin/gcc /usr/bin/gcc-snapshot
+
+# Install clang snapshot
+RUN wget -qO- https://apt.llvm.org/llvm-snapshot.gpg.key | tee /etc/apt/trusted.gpg.d/apt.llvm.org.asc && \
+    # Install add-apt-repository
+    apt-get update && apt-get install --no-install-recommends -y software-properties-common && \
```

Comment:
We don't seem to use `add-apt-repository`.

Reply:
Indeed, this was an old line. Removed.